packrat/src/test/java/com/github/reflectoring/infiniboard/packrat/source/SourceConfigTest.java

```diff
@@ -26,30 +24,36 @@
 public class SourceConfigTest {
 
     @Autowired
-    private SourceConfigRepository repository;
+    private SourceConfigRepository sourceConfigRepository;
+
+    @Autowired
+    private UrlSourceRepository urlSourceRepository;
+
+
+    private String testSourceConfigId;
 
     @Before
     public void setup() {
-        HashMap<String, Object> attributes = new HashMap<>();
-        attributes.put("alpha", "one");
-        attributes.put("beta", "two");
-        attributes.put("gamma", "three");
-        SourceConfig config = new SourceConfig("foo", "bar", new Date(), 5 * 60, attributes);
-
-        repository.save(config);
+        SourceConfig sourceConfig1 = new SourceConfig("widget1");
+
+        List<UrlSource> urlSources = new ArrayList<>();
+        urlSources.add(new UrlSource("www.foo1.bar", 2));
+        urlSources.add(new UrlSource("www.foo2.bar", 3));
+        urlSources.add(new UrlSource("www.foo3.bar", 6));
+        sourceConfig1.setUrlSources(urlSources);
+        urlSources = urlSourceRepository.save(urlSources);
+        sourceConfig1 = sourceConfigRepository.save(sourceConfig1);
+        testSourceConfigId = sourceConfig1.getId();
     }
 
     @Test
-    public void configWasSaved() {
-        List<SourceConfig> configList = repository.findBySourceId("bar");
-        assertThat(configList).hasSize(1);
+    public void sourceConfigSavedTest() {
```

Comment:
let's name our test methods with a pattern. anyone should be able to know what the test is expected to check by reading the test method name.

e.g.: findOneReturnsSavedSourceConfig

same problem in another PR: https://github.com/reflectoring/infiniboard/pull/38#discussion-diff-66727087R35